torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2660,14 +2624,6 @@ def specialize_zero_one(self):
     def duck_shape(self):
         return self.settings.duck_shape
 
-    @property
-    def prefer_deferred_runtime_asserts_over_guards(self):
-        return self.settings.prefer_deferred_runtime_asserts_over_guards
-
-    @property
-    def allow_complex_guards_as_runtime_asserts(self):
-        return self.settings.allow_complex_guards_as_runtime_asserts
-
     def check_equal(self, other: "ShapeEnv") -> None:
```

Comment:
Storing these flags in ShapeEnvSettings becomes a problem with AOTInductor, where the ShapeEnv is reused from exporting with Hybrid Symints = True, but later is frozen for runtime asserts for Inductor. We can't overwrite the settings, and this fails when we end up deferring another guard we want to keep.